core/src/main/java/com/datastax/oss/driver/internal/core/metadata/MetadataManager.java

```diff
@@ -106,7 +106,18 @@ private void onConfigChanged(@SuppressWarnings("unused") ConfigChangeEvent event
             || !keyspacesBefore.equals(refreshedKeyspaces)
             || (!tokenMapEnabledBefore && tokenMapEnabled))
         && isSchemaEnabled()) {
-      refreshSchema(null, false, true);
+      refreshSchema(null, false, true)
+          .whenComplete(
+              (metadata, error) -> {
+                if (error != null) {
+                  Loggers.warnWithException(
+                      LOG,
+                      "[{}] Unexpected error while refreshing schema after it was re-enabled "
```

Comment:
Nit: all call sites have `Unexpected error while...` except `CqlRequestHandler` which has `Error while...`.

Reply:
👍 I'll change this message.